src/redux/books/books.js

```diff
@@ -1,52 +1,77 @@
-// ACTIONS
-const ADD_BOOK = 'bookstore/books/ADD';
-const REMOVE_BOOK = 'bookstore/books/REMOVE';
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
Hello @prolajumokeoni when I use ReduxToolKit linters are showing errors because of the syntax. so is there anything I can do to avoid this?